charts/chainlink-cluster/Chart.lock

```diff
@@ -4,12 +4,12 @@ dependencies:
   version: 5.14.0
 - name: opentelemetry-collector
   repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-  version: 0.81.2
+  version: 0.82.0
```

Comment:
Should this be 0.95.0?

Reply:
ahh yep good catch!

Reply:
it should `0.82.0` since that's the latest chart version.